app/react/Metadata/components/MetadataFormFields.js

```diff
@@ -414,6 +414,8 @@ class MetadataFormFields extends Component {
 }
 
 MetadataFormFields.defaultProps = {
+  saveThesaurus: () => {},
```

Comment:
Both of these props are required, do they need defaults?

Reply:
I added this coz the linter was complaining that its undefined